official/utils/export/export_test.py

```diff
@@ -0,0 +1,61 @@
+# Copyright 2018 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Tests for exporting utils."""
```

Comment:
Would it be much trouble to put tests for the individual models? (Probably in their respective files rather than this one.) Basically generate a trivial model with synthetic data and then confirm that it can be loaded and serve.

Reply:
Hahahaha. I laugh because, as I learned during this process, actually using a saved model is incredibly difficult without a dedicated TF Serving instance. I am hoping we can change that next quarter, but, for now, it would require quite a few tf.contrib calls. Punting on that until we formalize the story on model exports, hopefully next quarter.